src/Printer.php

```diff
@@ -184,13 +184,13 @@ protected function writeException($exception)
     /**
      * Invoked when an exception is thrown in the test runner.
      *
-     * @param \Exception $e
+     * @param \Throwable|\Exception $t
      */
-    protected function onAddError(\Exception $e)
+    protected function onAddError($t)
```

Comment:
Can you not just type `\Throwable` (`\Exception` implements it anyway and it would be consistent with the property name/description)

Reply:
I can't because in theory this package should support older versions of PHP that don't have `Throwable`.